core/src/test/resources/Management/WebServices/get-services.json

```diff
@@ -2,7 +2,9 @@
     "services": [
         {
             "name": "TestRestListener",
-            "method": "GET",
+            "method": [
+                "GET"
+            ],
```

Comment:
Hier is ook een frontend aanpassing voor nodig.

Reply:
Ja, dat vermoedde ik ook en heb effe gekeken naar het Webservices scherm, maar die functioneerde nog gewoon, leek het. Heb je een suggestie waar ik precies moet zijn?

Reply:
Poeh, ik weet dat het eea in de `webservices.component.html` staat. Zo te zien mapt hij verder niets dus de data die de backend geeft kan je direct gebruiken.

Reply:
Ik denk dat ik het zo goed opgelost heb.